### PR TITLE
Updates for running full pipeline

### PR DESCRIPTION
**Pull request by gwarmstrong**
We should be able to run a full job now using something like ```scripts/run_optim.py```

### CODE REVIEW

File: scripts/run_optim.py [hunk 1]
```diff
@@ -1,33 +1,66 @@
...
-def run_optim(intial_subgraph, G, args):
+
+def run_optim(initial_subgraph, G, args):
 	if args.optim_type == 'randomized':
 		return optim.randomized_optimal_subgraph(initial_subgraph, G, args.k, args.alpha)
-	elif args.optim_type == 'simmulated_annealing':
+	elif args.optim_type == 'simulated_annealing':
 		return optim.simulated_annealing(initial_subgraph,
 										 G,
 										 args.alpha,
```
Comment (earmingol) [inline scripts/run_optim.py:66]: Add input for simulated_annealing: k_neighbors = args.k

**Comment by cameronmartino** [comment]
@gwarmstrong  Could you also fix issue #18 in this pull? Otherwise, looks good. thanks!

File: scripts/run_optim.py (hunk 1 above)
Comment (gwarmstrong) [reply scripts/run_optim.py:66]: This is already included see line 67

**Comment by earmingol** [comment]
@gwarmstrong I commited changes to optim.py in the contig_assembly branch to fix the overflow errors of simulated annealing. I would appreciate if you could include those changes here.

**Pull request merged**